src/test/java/nextstep/member/acceptance/AuthAcceptanceTest.java

```diff
@@ -26,17 +25,55 @@ class AuthAcceptanceTest {
     @Autowired
     private MemberRepository memberRepository;
 
-    @DisplayName("Bearer Auth")
     @Test
+    @DisplayName("Bearer Auth")
     void bearerAuth() {
         memberRepository.save(new Member(EMAIL, PASSWORD, AGE));
-
-        ExtractableResponse<Response> response = 로그인_토큰_요청(EMAIL, PASSWORD);
-        String accessToken = response.jsonPath().getString("accessToken");
+        var response = 로그인_토큰_요청(EMAIL, PASSWORD);
+        var accessToken = response.jsonPath().getString("accessToken");
         assertThat(accessToken).isNotBlank();
 
-        ExtractableResponse<Response> response2 = 본인_정보_조회(accessToken);
-
+        var response2 = 본인_정보_조회(accessToken);
         assertThat(response2.jsonPath().getString("email")).isEqualTo(EMAIL);
     }
+
+
+    /**
+     * Given 회원 가입을 사용자가
+     * When github 를 통해 인증을 요청하면
+     * Then 엑세스 토큰이 발행된다
+     */
+    @Test
+    @DisplayName("회원 가입을 한 사용자는 github 를 통해 엑세스 토큰을 발행할 수 있다.")
+    void githubAuthRequestFromMember() {
+        // when
+        var 사용자1 = GithubUsers.사용자1;
+        memberRepository.save(new Member(사용자1.getEmail(), PASSWORD, AGE));
+
+        // when
+        var response = Github_로그인_토큰_요청(사용자1.getCode());
+
+        // then
+        assertThat(response.jsonPath().getString("accessToken")).isNotBlank();
+    }
+
+    /**
+     * Given 회원 가입을 하지 않은 사용자가
+     * When github 를 통해 인증을 요청하면
+     * Then 엑세스 토큰이 발행된다
+     */
+    @Test
+    @DisplayName("회원 가입 하지 않은 사용자는 github 를 통해 가입과 동시에 엑세스 토큰을 발행할 수 있다.")
+    void githubAuthRequestFromNonMember() {
+        // given
+        var 사용자1 = GithubUsers.사용자1;
+        assertThat(memberRepository.findByEmail(사용자1.getEmail())).isEmpty();
```

Comment:
```suggestion
        // given
        var 회원_가입하지_않은_사용자 = GithubUsers.사용자1;
```

memberRepository.findByEmail은 없어도 될 것 같네요! 단, 회원 가입하지 않은 사용자라는 것을 표현해주면 좋아보여요!